### ISSUE TITLE
Cannot find any chromecast device?

### ISSUE THREAD

**Issue by room-homage** [philippe44/AirConnect#480]
Where do I start to debug such issue?

**Comment by philippe44** [comment]
it's usually because AirConnect is not on the same network/subnet as your CCs

**Comment by squallstar** [comment]
@philippe44 I have the same issue with a Apple Macbook Pro M2 Pro on Sonoma 14.0 (public beta).

I've got plenty of Chromecast compatible devices in my network and this is the only output I get from the script:

```
./airupnp-macos-arm64-static -d all=sdebug
[13:52:28.065201] main:1367 Starting airupnp version: v1.1.7 (May 20 2023 @ 06:12:13)
[13:52:28.065857] main:1374 no config file, using defaults
[13:52:28.066437] Start:1086 Binding to iface en0@192.168.1.202:0
[13:52:28.066553] bind_socket:636 socket binding 7 on port 52001
[13:52:28.066574] Start:1125 Starting pico HTTP server on port 52001
[13:52:58.004950] MasterHandler:560 event: 7 [UPNP_DISCOVERY_SEARCH_TIMEOUT] [0x0] (recurse 0)
[13:52:58.005058] UpdateThread:695 Presence checking
[13:53:28.004205] MasterHandler:560 event: 7 [UPNP_DISCOVERY_SEARCH_TIMEOUT] [0x0] (recurse 0)
[13:53:28.004258] UpdateThread:695 Presence checking
[13:53:35.493502] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.493925] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.494707] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.495503] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.496904] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.497688] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.498597] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.499280] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.501615] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.596054] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.698320] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.698746] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.700179] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.700975] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.701706] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.702487] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.703350] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.704020] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.706367] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:35.800941] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:53:58.004958] MasterHandler:560 event: 7 [UPNP_DISCOVERY_SEARCH_TIMEOUT] [0x0] (recurse 0)
[13:53:58.005090] UpdateThread:695 Presence checking
[13:54:16.965701] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:54:16.965701] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:54:16.966157] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:54:16.966757] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:54:16.967388] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:54:17.078196] MasterHandler:560 event: 4 [UPNP_DISCOVERY_ADVERTISEMENT_ALIVE] [0x0] (recurse 0)
[13:54:28.004921] MasterHandler:560 event: 7 [UPNP_DISCOVERY_SEARCH_TIMEOUT] [0x0] (recurse 0)
[13:54:28.005019] UpdateThread:695 Presence checking
```

**Comment by philippe44** [comment]
Old GenX speaking here 😄, it's not a script but a plain C/C++ compiled application. More seriously, you're using the version for UPnP devices, not the one for CC players.

**Comment by philippe44** [comment]
Should be fixed in 1.1.8

**Comment by philippe44** [comment]
Closed for inactivity
